src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -352,18 +352,42 @@ public async Task Build_Invalid_Single_Params_File_ShouldFail_WithExpectedErrorM
         }
 
         [TestMethod]
-        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/.*/main\.bicepparam")]
+        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/Extends/main\.bicepparam")]
         [TestCategory(BaselineHelper.BaselineTestCategory)]
-        public async Task Build_params_to_stdout_with_non_bicep_references_should_succeed(EmbeddedFile paramFile)
+        public async Task Build_params_to_stdout_with_extends_should_succeed(EmbeddedFile paramFile)
```

Comment:
Is it possible to avoid having a special-case test for extends? The `.*` wildcard in the `EmbeddedFilesTestData` attribute will intentionally pick up all files under the `Files/BuildParamsCommandTests` directory.

Reply:
I removed the redundant tests, since the files in `Extends` folder is already under-test 👍